patches/ios-chrome-app-main_controller.mm.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/ios/chrome/app/main_controller.mm b/ios/chrome/app/main_controller.mm
+index d776bc2c882434def3cf70afd40ea12edea3e2ca..d71322f9f61252dd5f600edc9f49143648418bd3 100644
+--- a/ios/chrome/app/main_controller.mm
++++ b/ios/chrome/app/main_controller.mm
+@@ -140,6 +140,9 @@
+ #import "ios/chrome/browser/credential_provider/model/credential_provider_service_factory.h"
+ #import "ios/chrome/browser/credential_provider/model/credential_provider_support.h"
+ #import "ios/chrome/browser/credential_provider/model/credential_provider_util.h"
++#endif
```

Comment:
where are we using main_controller.mm? Doesn't seem like that's something we would be including in the first place

Reply:
I'm not concerned about the patch specifically if it's being submitted to upstream, but I'm concerned about why this is being included in the ios build in the first place

Reply:
cc: @kylehickinson @cdesouza-chromium

Reply:
we pulled it in here, but it's not clear to me why https://github.com/brave/brave-core/pull/6722/files
I pinged @Brandon-T

Reply:
the dep on `app_internal` can be removed, it's not used anymore. When that is removed this patch will no longer be necessary

Reply:
@Brandon-T is working on this